### PR TITLE
Don't convert JSON arrays in output data to strings

### PR DESCRIPTION
**Pull request by gnestor**
@blink1073 The closest blame I could find was https://github.com/jupyterlab/jupyterlab/pull/1709/commits/75d5172443cac943b732db5489c5c2fde55fe63b. Can you confirm that this change won't break anything?

Before:

image

After:

image

### CODE REVIEW

**Comment by jasongrout** [comment]
Do you know why that check was there in the first place?

**Comment by gnestor** [comment]
No but I pinged @blink1073 to try to shed some light

**Comment by blink1073** [comment]
Looking again, display data should only be a string or a JSON object, so this change looks good.

File: packages/rendermime/src/outputmodel.ts [hunk 1]
```diff
@@ -345,9 +345,7 @@ namespace Private {
     for (let mimeType in bundle) {
       let item = bundle[mimeType];
       // Convert multi-line strings to strings.
```
Comment (jasongrout) [inline packages/rendermime/src/outputmodel.ts]: This comment can be removed now.

Comment (jasongrout) [reply packages/rendermime/src/outputmodel.ts]: Or perhaps you can change the comment to note that the purpose of the lines below is to deep-copy anything that isn't a primitive.

Comment (gnestor) [reply packages/rendermime/src/outputmodel.ts]: Is the purpose to deep-copy or parse stringified JSON? One other question: if `item` is a number, boolean, or null value, should we `JSON.parse` that, too?

Comment (jasongrout) [reply packages/rendermime/src/outputmodel.ts]: A number, boolean, string, or null value is immutable, so doing a copy doesn't help us. Notice that we first stringify, then parse it. Basically, this is an optimized way of doing a deep copy of some structure, i.e., a deep copy of an array or dict.

Comment (jasongrout) [reply packages/rendermime/src/outputmodel.ts]: (the input here is not a string containing JSON, the input here is an already-parsed JSON structure, i.e., `item` has already been through `JSON.parse`)

**Comment by gnestor** [comment]
Replacing this PR with https://github.com/jupyterlab/jupyterlab/pull/4480

**Pull request closed without merging**